docs/pbc/all/product-information-management/202410.0/base-shop/feature-overviews/product-comparison-feature-overview.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Product Comparison feature overview
+description: With the Product Bundles feature you can freely tie individual items together and sell them as a package.
+last_updated: Oct 24, 2024
+template: concept-topic-template
+---
+
+The Product Comparison feature lets your customers compare products based on their attributes.
+
+On the Product Details page, a customer can click **Compare** to add a product to comparison.
+
+![pdp-compare-button](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/product-information-management/base-shop/feature-overviews/product-comparison-feature-overview.md/pdp-compare-button.png)
+
+After a customer adds a second product to comparison, they're redirected to the **Compare products** page.
+
+![compare-products-page](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/product-information-management/base-shop/feature-overviews/product-comparison-feature-overview.md/compare-products-page.png)
+
+On this page, products are compared based on the attributes they have. The customer can remove individual or all products from comparison.
+
+A maximum of three products can be displayed by default.
```

Comment:
Please mention that this value can be changed in the configuration, see dev guide.